index.html

```diff
@@ -27,7 +27,117 @@ <h1>Charlie Giorgania</h1>
         </p>
         <button class="btn">Start Collaboration</button>
         <img class="combo" alt="" />
+        <h2>Projects</h2>
       </div>
     </section>
+
+    <section class="work-section">
+      <div class="grid-container">
+        <img class="mac" alt="" />
+        <img class="mac2" alt="" />
+        <img class="mac3" alt="" />
+        <img class="mac4" alt="" />
+        <img class="mac5" alt="" />
+        <img class="mac6" alt="" />
```

Comment:
- **[OPTIONAL]** I suggest using the images as requested by @adel-gu this will help you to fix the overlapping of the about section in the works section 👇🏽 

> [OPTIONAL] for better code structure and since the project item Image is part of the content, II highly recommend including each project img inside its own project grid item 🤔.